client/src/components/CustomerDetails.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { gql, useMutation } from '@apollo/client';
+
+import { useHistory } from "react-router-dom";
+
+const CUSTOMER_DETAILS = gql`
+    mutation EditCustomer($friendlyName:String! $phoneNumber:String!) {
+        createUser(friendlyName:$friendlyName, phoneNumber:$phoneNumber) {
+            id,
+            friendlyName,
+            phoneNumber 
+        }
+    }
+`;
+const CustomerDetails = () => {
+    const history = useHistory();
+    const manageCustomer = () => {
```

Comment:
this is an action that redirects to a page; you need a better name for that. Right now it's unclear what this function does.

Reply:
Ok, I will give a new name based on the name from the above question. like `redirectToCustomerListPage`. Is this name too long to you?

Reply:
Long variable names these days are no longer a big issue because most code editors have autocomplete features; but obviously you want to avoid long names because they are hard to read.

Reply:
I have changed this action's name to `backToLanding`, is this ok for you?

Reply:
where's the change? Those changes need to be in this PR.